src/Storage/Commands.Storage/File/Cmdlet/GetAzureStorageShare.cs

```diff
@@ -39,6 +40,14 @@ public class GetAzureStorageShare : AzureStorageFileCmdletBase
         public string Prefix { get; set; }
 
         [Parameter(
+        Position = 1,
+        Mandatory = false,
+        ParameterSetName = Constants.SpecificParameterSetName,
+        HelpMessage = "Snapshot Time of the file share snapshot to be gotten.")]
```

Comment:
Fix spacing

Reply:
Change to: "SnapshotTime of the file share snapshot to be gotten."